homework/12_B/09_Ivaylo_Georgiev/homework01/test_circles.py

```diff
@@ -0,0 +1,73 @@
+import math
+from msilib.schema import Binary
+from shutil import SameFileError
+from enum import Enum
+
+class Circle():
+    def __init__(self, x, y, radius):
+        self.x = x
+        self.y = y
+        self.radius = radius
+
+class Result(Enum):
+    Same = 1
+    B_in_A = 2
+    A_in_B = 3
+    Intersect = 4
+    Touch = 5
+    No_Touch = 6
+    
+c1 = Circle(-10, 8, 30)    
+c2 = Circle(14, -24, 10)
+c3 = Circle(-10, 8, 30) 
+c4 = Circle(8, 8, 8) 
+c5 = Circle(4, 3, 6)
+c6 = Circle(-4, 3, 6)
+
+def circle(c1, c2):
+    d = math.sqrt((c1.x - c2.x)**2 + (c1.y - c2.y)**2)
+    
+    #Проверка дали стойностите на двете окръжности са еднакви
+    if(c1.__dict__ == c2.__dict__):
+        print ("Circles are the same")
+        return Result.Same.value
+    elif(d <= c1.radius - c2.radius):
+        print("Circle B is inside A")
+        return Result.B_in_A.value
+    elif(d <= c2.radius - c1.radius):
+        print("Circle A is inside B")

```

Comment:
The prints should either be removed (if you added those for debug) or turned into a debug log.